app/controllers/account/cashes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Account::CashesController < ApplicationController
+  layout :with_sidebar
+
+  before_action :set_account
+
+  def index
+  end
+
+  private
+
+    def set_account
+      @account = Current.family.accounts.find(params[:account_id])
+    end
+end
```

Comment:
This controller is a "specialization" of `Account::HoldingsController`.  The "cash" section of an investment account will show:

1. Brokerage Cash - this is equal to `@account.balance` and represents the "brokerage cash" or "uninvested cash" being held in the investment account.  While brokerage cash is not technically a "holding", I've created a helper to simulate a `Holding` for displaying this row
  a. Sales of stock, dividends, interest accrued, and deposits (transfers from external account) increase brokerage cash
  b. Withdrawals from the account decrease brokerage cash
2. Cash Equivalents - in the future, we may show a list of cash-equivalent holdings here (e.g. USDC, money market funds, etc.)